### ISSUE TITLE
Clarify Evaluation in config file of the model (metrics)

### ISSUE THREAD

**Issue by Petros626** [tensorflow/models#10703]
Hi guys,

I have seen that the use of m**odel_main.py** is recommended, as it offers the advantage of not having to work with train.py and then eval.py.

However, I have a question, because the first results seemed strange to me (maybe its too early). 
I use images, but these are images of point cloud data displayed as a bird's eye view.

I have the following questions:
1. Does model_main.py script evaluate my dataset against the COCO dataset?
2. How can I evaluate my dataset against my ground truth boxes?

The results seem really bad to me, the number of my test images are 1051. It's not clear for me against what my test images are evaluated, if against my ground truth labels fine, but if against COCO it doesn't make sense and I need help to change that

![IMG_20220712_114116](https://user-images.githubusercontent.com/62354721/178464508-57be563d-7246-43e0-a832-919c75aaa23d.jpg)

**Comment by Petros626** [comment]
@jvishnuvardhan could you explain it to me? I'm not sure, if my own data are evuluated (ground truth vs. predicted) or it' s using always COCO, which would be sad.

**Comment by sushreebarsa** [comment]
@Petros626 
In order to expedite the trouble-shooting process, could you please provide the entire URL of the repository which you are using. Please provide more details on the issue reported here.Thank you!
